src/test/java/org/springframework/data/couchbase/util/UnmanagedTestCluster.java

```diff
@@ -69,8 +69,9 @@ TestClusterConfig _start() throws Exception {
 						.build())
 				.execute();
 
-		if (postResponse.code() != 202) {
-			throw new Exception("Could not create bucket: " + postResponse + ", Reason: " + postResponse.body().string());
+		String reason = postResponse.body().string();
+		if (postResponse.code() != 202 && !(reason.contains("Bucket with given name already exists"))) {
```

Comment:
this shouldn't exist with the uuid change above, right?

Reply:
When reusing "my_bucket", 202(created) and "already exists" is acceptable, don't throw an exception.